src/intervals/arithmetic.jl

```diff
@@ -375,15 +375,21 @@ Assumes 0 ≤ α ≤ 1.
 function mid(a::Interval{T}, α) where T
 
     isempty(a) && return convert(T, NaN)
-    isentire(a) && return zero(a.lo)
+
+    if isentire(a)
+        α == 0.5 && return zero(a.lo)
+        # Shift center if α != 0.5
+        lo = nextfloat(-∞)
+        hi = prevfloat(+∞)
+        return (1-α) * lo + α * hi
```

Comment:
I wonder if this should be something like a geometric mean?

Reply:
The use case I have in mind (shifted midpoint for `IntervalRootFinding`) only requires something like

```julia
mid(-Inf..Inf, 0.4) < 0
```

so the average used does not really matter. Clearly using a geometric mean makes more sense, but I thing that expecting anything really meaningful when calling `mid(-Inf..Inf, α)` would be very optimistic from the user.

Also using different types of average for different inputs may be confusing.